drivers/hmis/spec/requests/hmis/ce/assign_referral_participants_spec.rb

```diff
@@ -0,0 +1,189 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require_relative '../../../support/ce_spec_helper'
+
+RSpec.describe Mutations::Ce::AssignReferralParticipants, type: :request do
+  include_context 'ce spec helper'
+
+  before(:each) do
+    hmis_login(user)
+  end
+
+  let!(:provider_swimlane) { workflow_template.swimlanes.create!(name: 'Providers') }
+  let!(:provider_acceptance_task) do
+    create(
+      :hmis_workflow_definition_task,
+      template: workflow_template,
+      name: 'Provider Acceptance',
+      swimlane: provider_swimlane,
+      form_definition: create(:hmis_form_definition),
+    )
+  end
+
+  let!(:hmis_user2) { create(:hmis_user, data_source: ds1) }
+
+  describe 'assign participants mutation' do
+    let(:mutation) do
+      <<~GRAPHQL
+        mutation AssignParticipants($referralId: ID!, $participants: [CeReferralParticipantInput!]!) {
+          assignReferralParticipants(referralId: $referralId, participants: $participants) {
+            referral {
+              id
+              swimlanes {
+                id
+                name
+                participants {
+                  id
+                  name
+                }
+              }
+            }
```

Comment:
From the frontend, we also resolve:
```
      steps {
          id
          stepId
          name
          status
          swimlane
          assignees {
            id
            name
          }
      }
```
It's probably worth adding those here just so we know that the query we're actually performing is able to fully resolve (aka doesn't crash). I see you're already testing the presence of the assignees on the step, so I think the assertions are fine as-is, but it can help future-bug-catching to resolve a more realistic payload.